File: Logical/2.rb
puts("아이디를 입력하세요\n")
id_str = gets.chomp()

puts("비밀번호를 입력하세요")
pw_str = gets.chomp()
id_real = "egoing"
pw_real = "1234"

if id_real == id_str
  if pw_real == pw_str
    puts("Hello  " + id_real + " 님")
  else
    puts("패스워드를 잘못 입력하셨어요.")
  end
else
  puts("아이디를 잘못 입력하셨어요.")
end

File: Logical/2.py
id_str = input("아이디를 입력하세요.\n")
pw_str = input("비밀번호를 입력하세요\n")
id_egoing = "egoing"
pw_egoing = "1234"

if (id_egoing == id_str):
    if (pw_egoing == pw_str):
        print("Hello!" + id_str + " 님")
    else:
        print("비밀번호가 틀렸습니다.")
else:
    print("Who are you?")
